### DISCUSSION TITLE
Importing midi files?

### DISCUSSION THREAD

**Discussion by plasticbutterknife** [titinko/utsu#80] [Ideas]
My suggestion is to make it so that you can import midi files into Utsu. Otherwise, I wouldn't really have any way to make my own usts (that are not incredibly painful to do) since Utau-Synth doesn't work with Mac Big Sur. You don't have to, but it would be nice. Thanks!

**Comment by titinko** [comment]
See the open bug for MIDI files: https://github.com/titinko/utsu/issues/43

**Reply by plasticbutterknife** [reply]
Alright, thanks!
